pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -841,12 +841,37 @@ func describeAdditionalBuildDetail(build *buildgraph.BuildConfigNode, lastSucces
 		lastTime = passTime
 	}
 
+	var firstBuildToDisplay *buildgraph.BuildNode
+	firstTime := unversioned.Time{}
+	var secondBuildToDisplay *buildgraph.BuildNode
+	secondTime := unversioned.Time{}
+
 	// display the last successful build if specifically requested or we're going to display an active build for context
 	if lastSuccessfulBuild != nil && (includeSuccess || len(activeBuilds) > 0) {
-		out = append(out, describeBuildPhase(lastSuccessfulBuild.Build, &passTime, build.BuildConfig.Name, pushTargetResolved))
```

Comment:
do you actually need to check if lastSuccessBuild!=nil?  seems like all this logic works fine even if "firstBuildToDisplay" or "secondBuildToDisplay" ends up getting assigned "nil"